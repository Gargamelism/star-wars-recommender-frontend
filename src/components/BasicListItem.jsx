import React from "react";

export default function BasicListItem({ name }) {
    return (
        <span key={name} className="flex-grow text-center border-2 rounded border-gray-400 m-1 px-1">{name}</span>
    );
}