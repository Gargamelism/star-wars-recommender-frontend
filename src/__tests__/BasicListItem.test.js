import renderer from "react-test-renderer";
import BasicListItem from "../components/BasicListItem";

test("renders BasicListItem", () => {
  const tree = renderer
    .create(<BasicListItem name="123" />)
    .toJSON();
    
    expect(tree).toMatchSnapshot();  
});
