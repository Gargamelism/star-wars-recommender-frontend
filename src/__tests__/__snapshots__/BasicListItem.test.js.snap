// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders BasicListItem 1`] = `
<span
  className="flex-grow text-center border-2 rounded border-gray-400 m-1 px-1"
>
  123
</span>
`;
